runtime/wasm/src/host_exports.rs

```diff
@@ -246,7 +246,15 @@ where
         &self,
         bytes: Vec<u8>,
     ) -> Result<String, HostExportError<impl ExportError>> {
-        let s = String::from_utf8(bytes).map_err(HostExportError)?;
+        let s = String::from_utf8(bytes).map_err(|e| {
+            HostExportError(format!(
+                "Failed to parse byte array using `toString()`. This may be caused by attempting \
+                 to convert a value such as an address that cannot be parsed to a unicode string. \
+                 Try 'toHexString()' instead. Bytes:`{bytes:?}`. Error: {error}",
```

Comment:
There's a space missing after `Bytes:`.

Reply:
@Jannis Updated! Thanks for noticing the spacing. 🥇